src/static/js/controllers.js

```diff
@@ -961,7 +962,7 @@ app.controller("edit-nc", function($scope, dataFactory, $http, $routeParams, $wi
     $scope.id = 0;
     $scope.submit = function(user_id, nc_id){
         if(true){
-            dataFactory.put('/users/'+user_id,{'name' : $scope.user,'email' : $scope.email} ).
+            dataFactory.put('/users/'+user_id,{'name' : $scope.user, 'email' : $scope.email, 'phone' : $scope.phone} ).
```

Comment:
POST Method also should have 'phone' field. In case of adding a new Nodal Coordinate.